client/components/LearnerViewDirective.js

```diff
@@ -45,6 +45,12 @@ tie.directive('learnerView', [function() {
                   {{paragraph.getContent()}}
                 </p>
               </div>
+              <div class="tie-dot-container"
+                   ng-class="{'tie-dot-hidden': hideDotDiv}">
```

Comment:
If the only point of this is to hide/show, then how about using ng-if="loadingIndicatorIsShown" instead? Using ng-if seems simpler since it's already built-in. The reason for the name change is twofold:

- the "concept" behind the doc is a loading indicator. Its visual appearance may change in the future.
- hideDotDiv starts with a verb, and therefore sounds like a method/function.